### PR TITLE
Fix scan command

### PR DESCRIPTION
**Pull request by kapbh**

### CODE REVIEW

**Review by krish2718 — APPROVED** [review]

File: subsys/net/l2/wifi/wifi_shell.c [hunk 1]
```diff
@@ -1824,9 +1824,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
...
+		    "[-s, --ssid] : SSID to scan for. Can be provided multiple times\n"
 		    "[-m, --max_bss <val>] : Maximum BSSes to scan for. Range 1 - 65535\n"
-		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6-11,14_5:36,149-165,44\n"
+		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6_5:36\n"
```
Comment (rado17) [inline subsys/net/l2/wifi/wifi_shell.c:1829]: It would be better if we cover channel range (`-`) as well, in this example.

**Review by rado17 — CHANGES_REQUESTED** [review]

File: subsys/net/l2/wifi/wifi_shell.c (hunk 1 above)
Comment (kapbh) [reply subsys/net/l2/wifi/wifi_shell.c:1829]: Ok. I will update multiple examples for this.

File: subsys/net/l2/wifi/wifi_shell.c [hunk 2]
```diff
@@ -1824,9 +1824,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
...
+		    "[-s, --ssid] : SSID to scan for. Can be provided multiple times\n"
 		    "[-m, --max_bss <val>] : Maximum BSSes to scan for. Range 1 - 65535\n"
-		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6-11,14_5:36,149-165,44\n"
+		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6_5:36 or 2:1,6-11,14_5:36,163-177,52\n"
```
Comment (rado17) [inline subsys/net/l2/wifi/wifi_shell.c:1829]:
```suggestion
		    "[-c, --chans <Comma separated list of channel ranges>] : Channels to be scanned. The channels must be specified in the form band1:chan1,chan2_band2:chan3,..etc. band1, band2 must be valid band values and chan1, chan2, chan3 must be specified as a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. For example, a valid channel specification can be 2:1,6_5:36 or 2:1,6-11,14_5:36,163-177,52. Care should be taken to that configured channels don't exceed CONFIG_WIFI_MGMT_SCAN_CHAN_MAX_MANUAL\n"
```

**Review by rado17 — CHANGES_REQUESTED** [review]

File: subsys/net/l2/wifi/wifi_shell.c (hunk 2 above)
Comment (sachinthegreen) [reply subsys/net/l2/wifi/wifi_shell.c:1829]: Is an `ensure` missing in the sentence? :

`Care should be taken to that configured channels don't exceed CONFIG_WIFI_MGMT_SCAN_CHAN_MAX_MANUAL`

Comment (kapbh) [reply subsys/net/l2/wifi/wifi_shell.c:1829]: Yes. I have raised the upstream PR with correction. waiting for the merge.

**Review by rado17 — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**